akka-sse/src/main/scala/de/heikoseeberger/akkasse/ServerSentEvent.scala

```diff
@@ -21,6 +21,12 @@ import scala.annotation.tailrec
 
 object ServerSentEvent {
 
+  sealed trait EventIdValue
+  case object ResetId extends EventIdValue
+  final case class Id(id: String) extends EventIdValue {
```

Comment:
Then we could not use the require in the constructor. Should we drop validation then?